<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hudExp</class>
 <widget class="QWidget" name="hudExp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>276</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="0" column="3">
      <widget class="QPushButton" name="btnExpResetCurrent">
       <property name="text">
        <string>Reset Current</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="btnExpSaveLibrary">
       <property name="text">
        <string>Save Library</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnExpMode">
       <property name="text">
        <string>Mode (ALL)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lblTotalExp">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblTotalExpLabel">
       <property name="text">
        <string>Total Exp:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0" colspan="6">
      <widget class="QTableView" name="tblExpLookup">
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
